packages/pytest-simcore/src/pytest_simcore/aws_server.py

```diff
@@ -73,6 +77,42 @@ def mocked_ec2_server_envs(
     return setenvs_from_dict(monkeypatch, changed_envs)
 
 
+@pytest.fixture
+def with_patched_ssm_server(
+    mocker: MockerFixture, external_envfile_dict: EnvVarsDict
+) -> mock.Mock:
+    if external_envfile_dict:
+        # NOTE: we run against AWS. so no need to mock
+        return mock.Mock()
+    return mocker.patch(
+        "aiobotocore.client.AioBaseClient._make_api_call",
+        side_effect=patched_aiobotocore_make_api_call,
+        autospec=True,
+    )
+
+
+@pytest.fixture
+def mocked_ssm_server_settings(
+    mocked_aws_server: ThreadedMotoServer,
+    with_patched_ssm_server: mock.Mock,
+    reset_aws_server_state: None,
+) -> SSMSettings:
+    return SSMSettings(
+        SSM_ACCESS_KEY_ID="xxx",
+        SSM_ENDPOINT=f"http://{mocked_aws_server._ip_address}:{mocked_aws_server._port}",  # pylint: disable=protected-access # noqa: SLF001
+        SSM_SECRET_ACCESS_KEY="xxx",  # noqa: S106
+    )
+
+
+@pytest.fixture
+def mocked_ssm_server_envs(
+    mocked_ssm_server_settings: SSMSettings,
+    monkeypatch: pytest.MonkeyPatch,
+) -> EnvVarsDict:
+    changed_envs: EnvVarsDict = mocked_ssm_server_settings.dict()
```

Comment:
nice. Yes, this workflow verifies everything circularly.   settings -> env vars -> capture back as settings